content/pt-br/docs/reference/access-authn-authz/abac.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Utilizando Autorização ABAC
+content_type: concept
+weight: 80
+---
+
+<!-- overview -->
+O controle de acesso baseado em atributos (ABAC) define um paradigma de controle de acesso onde os direitos de acesso são concedidos aos usuários por meio do use de políticas que combinam atributos.
+
+<!-- body -->
+## Formato do arquivo de política
+
+Especifique os parametros de inicialização `--authorization-policy-file=NOME_DE_ALGUM_ARQUIVO` e `--authorization-mode=ABAC` para habilitar o modo `ABAC`.
+
+O formato do arquivo é de [um objeto JSON por linha](https://jsonlines.org/). Nele não deve haver lista ou mapa envolvente, apenas um mapa por linha.
+
+Cada linha é um "objeto de política", onde cada objeto é um mapa com as seguintes propriedades:
+
+  - Propriedades de versionamento:
```

Comment:
The list indentation needs some review. Some items are excessively indented while others are not indented enough:
![Portuguese version rendering](https://github.com/kubernetes/website/assets/698465/a52fa56a-b541-4dc9-9506-8160e8b34d2d)
![English version rendering](https://github.com/kubernetes/website/assets/698465/b17e6498-9bc6-4007-b230-9b3bacc0ec85)

Reply:
Tried to addressed in [929685e](https://github.com/kubernetes/website/pull/41800/commits/929685eb8a05d2319710c87419e0da1829aef706), let me know if I still left something out.

Reply:
Yes, this is solved. Thank you!